frontend/src/clients/newHttpClient.ts

```diff
@@ -0,0 +1,17 @@
+import axios, { AxiosInstance } from 'axios'
+
+export class HttpClient {
+  protected httpTimeout = 300000
+  protected axiosInstance: AxiosInstance
+
+  constructor() {
+    this.axiosInstance = axios.create({
+      baseURL: '/api/v1',
+      timeout: this.httpTimeout,
+    })
+    this.axiosInstance.interceptors.response.use(
```

Comment:
There seems to be no need to add interceptor

Reply:
make sense.